### PR TITLE
Chanege readme format from adoc to md.

### PR DESCRIPTION
**Pull request by pkliczewski**
Let's be consistent with other ManageIQ project and use markdown instead of adoc as README.

### CODE REVIEW

**Comment by pkliczewski** [comment]
@jhernand please review

File: README.md [hunk 1]
```diff
@@ -9,32 +9,21 @@
...
-
-* For the `manageiq` project:
-+
-https://github.com/jhernand/manageiq/tree/kubevirt_provider
-
-* For the `manageiq-ui-classic` project:
-+
-https://github.com/jhernand/manageiq-ui-classic/tree/kubevirt_provider
-
-== Things that work
```
Comment (jhernand) [inline README.md]: This should also be changed to `##`.

File: README.md [hunk 2]
```diff
@@ -86,33 +68,41 @@ details. In addition Kubernetes itself doesn't yet support sub-resources
...
 
-== Notes
+## Notes
 
-=== How to get the default token from Kubernetes
+### How to get the default token from Kubernetes
 
 List the set of secrets:
 
+  ```
```
Comment (jhernand) [inline README.md]: For code blocks you can just change the indentation from 2 spaces to 4 spaces. If you are going to use the three backticks instead, then I'd suggest that you remove the indentation completely.

Comment (pkliczewski) [reply README.md]: I checked this and it seems not to work so I added the line that you commented.

File: README.md (hunk 1 above)
Comment (pkliczewski) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pkliczewski) [reply README.md]: Here is [code quoting](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code) guideline.

File: README.md [hunk 3]
```diff
@@ -9,33 +9,22 @@
 [![Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ManageIQ/manageiq-providers-kubevirt?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Translate](https://img.shields.io/badge/translate-zanata-blue.svg)](https://translate.zanata.org/zanata/project/view/manageiq-providers-kubevirt)
 
-This repository contains an experimental provider that integrates the
-http://manageiq.org[ManageIQ] and https://github.com/kubevirt[KubeVirt]
-projects.
+Experimental [ManageIq](https://github.com/ManageIQ) provider which integrates 
+[Kubevirt](https://github.com/kubevirt) project.
```
Comment (masayag) [inline README.md]: please merge lines

File: README.md [hunk 4]
```diff
@@ -9,33 +9,22 @@
...
-* For the `manageiq-ui-classic` project:
-+
-https://github.com/jhernand/manageiq-ui-classic/tree/kubevirt_provider
-
-== Things that work
+## Things that work
 
 Currently the provider supports the following simple use cases:
 
 1. Add the provider using token athentication.
```
Comment (masayag) [inline README.md]: typo - athentication

File: README.md [hunk 5]
```diff
@@ -86,33 +68,41 @@ details. In addition Kubernetes itself doesn't yet support sub-resources
...
 
+  ```
   $ kubectl get secrets
   NAME                  TYPE                                  DATA      AGE
   default-token-7psxt   kubernetes.io/service-account-token   3         20d
+  ```
 
 Get the details of the `default` token:
 
+  ```
```
Comment (masayag) [inline README.md]: might not be part of this PR, but we can simplify the steps below by using:

`kubectl describe secret default-token-7psxt`

So decoding the token is not required.

File: README.md (hunk 4 above)
Comment (pkliczewski) [reply README.md]: done
